modules/atomic-widgets/dynamic-tags.php

```diff
@@ -5,21 +5,34 @@
 use Elementor\Modules\AtomicWidgets\Controls\Section;
 use Elementor\Modules\AtomicWidgets\Controls\Types\Select_Control;
 use Elementor\Modules\AtomicWidgets\Controls\Types\Text_Control;
+use Elementor\Modules\AtomicWidgets\PropTypes\Dynamic_Prop_Type;
 use Elementor\Modules\AtomicWidgets\PropTypes\Image_Prop_Type;
 use Elementor\Modules\AtomicWidgets\PropTypes\Number_Prop_Type;
+use Elementor\Modules\AtomicWidgets\PropTypes\Prop_Type;
 use Elementor\Modules\AtomicWidgets\PropTypes\String_Prop_Type;
 use Elementor\Modules\DynamicTags\Module as Dynamic_Tags_Module;
 
 class Dynamic_Tags {
 	public function register_hooks() {
-		add_filter( 'elementor/editor/localize_settings', fn( $settings ) => $this->add_atomic_dynamic_tags_settings( $settings ) );
+		add_filter(
+			'elementor/editor/localize_settings',
+			fn( $settings ) => $this->add_atomic_dynamic_tags_settings( $settings )
+		);
```

Comment:
Is it required?

Reply:
we still need the actual tags configuration in the frontend